### PR TITLE
[REF] hr_timesheet: convert TaskWithHours to owl

### PR DESCRIPTION
**Pull request by IT-Ideas**
task-2936932

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_timesheet/static/src/components/timesheet_uom/timesheet_uom.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    get FactorCompanyDependentProps() {
+        const factor = this.companyService.currentCompany.timesheet_uom_factor || this.props.factor;
+        return { ...this.props, factor };
+    }
+
+}
+
+TimesheetUOM.template = "hr_timesheet.TimesheetUOM";
```
Comment (xavierbol) [inline addons/hr_timesheet/static/src/components/timesheet_uom/timesheet_uom.js:47]: No need to define the props? 🤔

**Review by xavierbol — COMMENTED** [review]
I have just one comment, after that it will look good to me. 🙂

File: addons/hr_timesheet/static/src/components/timesheet_uom/timesheet_uom.js (hunk 1 above)
Comment (IT-Ideas) [reply addons/hr_timesheet/static/src/components/timesheet_uom/timesheet_uom.js:47]: 🙈

**Comment by xavierbol** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**